packages/dates/src/tests/is-less-than-one-minute-ago.test.ts

```diff
@@ -1,4 +1,5 @@
 import {clock} from '@shopify/jest-dom-mocks';
+
```

Comment:
Hmm I am pretty surprised we are enforcing this.

Reply:
Its not something I would have added, but if people want it I think its fine.